specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/types/applications.json

```diff
@@ -0,0 +1,365 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Application types",
+    "version": "2023-01-31-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "application": {
+      "type": "object",
+      "description": "Application tracked resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "description": "The system metadata of the application resource.",
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        },
+        "properties": {
```

Comment:
TrackedResource from v3 common types already includes systemData. You should remove this extra/redundant copy.

Reply:
Removed the redundant systemData from the tracked resource properties.

Reply:
removed the redundant systemData